src/components/transport_manager/test/transport_manager_default_test.cc

```diff
@@ -80,11 +82,13 @@ TEST(TestTransportManagerDefault, Init_LastStateNotUsed) {
   transport_manager::TransportManagerDefault transport_manager(
       transport_manager_settings);
 
-  NiceMock<MockLastState> mock_last_state;
+  auto mock_last_state = std::make_shared<MockLastState>();
```

Comment:
@YarikMamykin maybe it makes more sense to create MockLastState as the member of `TestTransportManagerDefault`?

Reply:
@AKalinich-Luxoft fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/fbb80545f327f8a8bdae59ccd540dc284e7e8441